packages/perennial/contracts/multiinvoker/MultiInvoker.sol

```diff
@@ -68,43 +68,48 @@ contract MultiInvoker is IMultiInvoker, UInitializable {
      * @param invocations The list of invocations to execute in order
      */
     function invoke(Invocation[] calldata invocations) external {
+        // ACTION TYPES
+        // 
+        // 0000 0000
+        // 
+
         for (uint256 i = 0; i < invocations.length; i++) {
             Invocation memory invocation = invocations[i];
 
             // Deposit from `msg.sender` into `account`s `product` collateral account
             if (invocation.action == PerennialAction.DEPOSIT) {
                 (address account, IProduct product, UFixed18 amount) = abi.decode(invocation.args, (address, IProduct, UFixed18));
-                depositTo(account, product, amount);
+                _depositTo(account, product, amount);
```

Comment:
nit: I think it's a little cleaner to naming these the same as the `Action`, so like `_deposit()` for this one.